### ISSUE TITLE
Configuration of a MathInput component

### ISSUE THREAD

**Issue by ddoyen** [PremierLangage/premierlangage#309]
I have yet another issue with the configuration of MathInput components (after #302).

I wanted to disable the _smartFence_ mode, which adds automatically a ghost right parenthesis when one types a left parenthesis. Here is what I declared in the PL file.

~~~
input1.config %=
{
    "inlineShortcuts" : { "*": "\\times"},
    "smartFence": false
}
==
~~~

The inline shortcut is indeed added, but the _smartFence_ mode is not disabled. 

Here is the link to the whole PL file : https://pl.u-pem.fr/filebrowser/option?name=test_pl&path=Yggdrasil/Mathematics/exercises/testcomp/test2.pl

Here is how the component should behave : https://pl.u-pem.fr/filebrowser/option?name=test_pl&path=Yggdrasil/Mathematics/exercises/testcomp/test.pl

**Comment by mciissee** [comment]
I think this is a problem with the version of MathLive that components use. In your version you use the latest version of MathLive by using a CDN link but in components I use a specific version for security reason (I don't want that a new version of MathLive break the code of the component).

I will try to update the version to check if the problem still occurs.

**Comment by mciissee** [comment]
Moved to components repository so this one will be closed
